engine/src/flutter/tools/licenses_cpp/data/exclude.txt

```diff
@@ -8,112 +8,118 @@
 .*/tests/.*
 .*third_party/googletest/.*
 ^build/.*
-^buildtools/.*
-^impeller/fixtures/.*
-^prebuilts/.*
-^shell/platform/fuchsia/dart-pkg/zircon_ffi/lib/zircon_ffi.dart
-^sky/packages/sky_engine/LICENSE
-^testing/.*
-^third_party/android_tools/.*
-^third_party/angle/third_party/.*
-^third_party/angle/tools/.*
-^third_party/angle/util/.*
-^third_party/benchmark/.*
-^third_party/dart/pkg/.*
-^third_party/dart/third_party/binary_size/.*
-^third_party/dart/third_party/binaryen/.*
-^third_party/dart/third_party/d3/.*
-^third_party/dart/third_party/devtools/web/.*\.frag$
-^third_party/dart/third_party/devtools/web/.*\.js$
-^third_party/dart/third_party/pkg/.*
-^third_party/dart/third_party/requirejs/.*
-^third_party/depot_tools/.*
-^third_party/expat/expat/lib/internal\.h$
-^third_party/freetype2/docs/[^F].*$
-^third_party/google_fonts_for_unit_tests/.*
-^third_party/gradle/.*
-^third_party/harfbuzz/.*\.py
-^third_party/harfbuzz/perf/.*
-^third_party/harfbuzz/src/harfbuzz.cc
-^third_party/harfbuzz/util/.*
-^third_party/icu/scripts/.*
-^third_party/imgui/.*
-^third_party/inja/third_party/amalgamate/.*
-^third_party/inja/third_party/include/doctest/.*
-^third_party/java/.*
-^third_party/libpng/contrib/.*
-^third_party/libwebp/.*\.py
-^third_party/libwebp/examples/.*
-^third_party/libwebp/extras/webp_quality.c
-^third_party/libwebp/swig/.*
-^third_party/llvm-project/.*
-^third_party/ocmock/.*
-^third_party/pkg/archive/.*
-^third_party/pkg/flutter_packages/.*
-^third_party/pkg/gcloud/.*
-^third_party/pkg/googleapis/.*
-^third_party/pkg/process/.*
-^third_party/pkg/vector_math/.*
-^third_party/protobuf/.*
-^third_party/pyyaml/.*
-^third_party/skia/third_party/freetype2/.*
-^third_party/skia/third_party/icu/.*
-^third_party/skia/third_party/libgrapheme/.*
-^third_party/skia/third_party/libpng/.*
-^third_party/skia/third_party/vello/.*
-^third_party/swiftshader/third_party/.*
-^third_party/swiftshader/third_party/llvm-10.0/.*
-^third_party/swiftshader/third_party/llvm-16.0/.*
-(?i)^third_party/vulkan-deps/glslang/src/LICENSE\.txt$
-^third_party/vulkan-deps/spirv-headers/.*
-^third_party/vulkan-deps/vulkan-headers/src/LICENSE\.md$
-^third_party/vulkan-deps/vulkan-loader/.*
-^third_party/vulkan-deps/vulkan-utility-libraries/src/LICENSE.md$
-^third_party/vulkan-deps/vulkan-utility-libraries/src/scripts/.*
-^tools/android_sdk/packages\.txt$
-^tools/fuchsia/.*
-^tools/licenses_cpp/data/.*
+^build_overrides/.*
+^fuchsia/.*
+^out/.*
```

Comment:
no action required: Unfortunately the way the license checker is designed is to be auditable.  So even though we don't parse any of these files, just looking at them causes a cost in disk access and logs.  This is going to make the logs much harder to audit.